### DISCUSSION TITLE
specify the number of output items, and sorting

### DISCUSSION THREAD

**Discussion by lxl66566** [dvershinin/lastversion#138] [Ideas]
Is there any way to specify the number of output assets?
If no, how about adding a `--number`(or `--num-assets`) option?

For example,
```
lastversion eza-community/eza --assets
https://github.com/eza-community/eza/releases/download/v0.17.0/eza_x86_64-unknown-linux-gnu.tar.gz
https://github.com/eza-community/eza/releases/download/v0.17.0/eza_x86_64-unknown-linux-gnu.zip
https://github.com/eza-community/eza/releases/download/v0.17.0/eza_x86_64-unknown-linux-musl.tar.gz
https://github.com/eza-community/eza/releases/download/v0.17.0/eza_x86_64-unknown-linux-musl.zip
```
but only one is needed. (any one is ok)
And sorting is for someone that prefers musl than gnu, (maybe?)

The reason why not use filter is that some releases may not provide `tar.gz` as well as `musl`, and some may only provide `tar.gz`.

**Comment by dvershinin** [comment]
There's already too many possible cli switches to lastversion that I think in this case you can simply pipe its output to xargs and sort programs which are universally available
